registry.yaml

```diff
@@ -344,7 +343,6 @@
   constraints: ">=0.1.0"
   products:
     - oss
-    - cloud
```

Comment:
These two SQL drivers were removed by Daniel in the reference doc: https://docs.google.com/document/d/1QiBXYFaFGhnrXaEGqDoKqmmAplaP7lf__UquvWQdyB0/edit?tab=t.0#heading=h.vlyf0dymhww